ghconfig.py

```diff
@@ -80,6 +80,19 @@ def remove_collaborator(organisation, repository, username):
     return response.status_code
 
 
+def remove_collaborator_from_organisation(organisation, username):
```

Comment:
Perhaps this should be named
`remove_member_from_organisation` 

Collaborators have some specific meaning in github (attached to a repo not an organisation)